src/GraphQL/Execution/ErrorInfoProvider.cs

```diff
@@ -133,30 +132,13 @@ private static string NormalizeErrorCode(Type exceptionType)
                 code = code.Substring("GraphQL".Length);
             }
 
-            return GetAllCapsRepresentation(code);
-        }
-
-        private static string GetAllCapsRepresentation(string str)
-        {
-            return Regex
-                .Replace(NormalizeString(str), @"([A-Z])([A-Z][a-z])|([a-z0-9])([A-Z])", "$1$3_$2$4")
```

Comment:
It's funny. I saw this place many times but did not think to change it to `ToConstantCase`.